p2p_wallet/Scenes/Main/Send/RecipentSearch/RecipientSearchViewModel.swift

```diff
@@ -241,17 +243,17 @@ class RecipientSearchViewModel: ObservableObject {
     
     // MARK: - Send via link
     
-    func checkIfSendViaLinkAvailable() async throws {
```

Comment:
we needs to get limit from server, so `checkIfSendViaLinkAvailable` must be async throws

Reply:
but now we don't use. Why do we need to use async throws before server implementation?

Reply:
when we implement server part, we'll add async throws, it's better practice